src/hist.jl

```diff
@@ -259,34 +272,48 @@ end
 append!{T,N}(h::AbstractHistogram{T,N}, vs::NTuple{N,AbstractVector}, wv::WeightVec) = append!(h, vs, values(wv))
 
 
+# Turn kwargs nbins into a type-stable tuple of integers:
+function _nbins_tuple{N}(vs::NTuple{N,AbstractVector}, nbins)
+    template = map(length, vs)
+
+    if VERSION < v"0.6.0-"
```

Comment:
Is this Ok? I don't know at what point Julia v0.6 started to support broadcast for tuples.